docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+services:
+    web:
+        hostname: web
+        build: .
+        image: libreramboxdeploy/identihub
+        ports:
+            - 8080:80
+        links:
+            - superdb
+        networks:
+            - external_network
+            - internal_network
+        # volumes:
+    superdb:
+        hostname: superdb
+        image: mysql
+        networks:
+            - internal_network   
+        environment:
+            MYSQL_RANDOM_ROOT_PASSWORD: "yes"
+            MYSQL_DATABASE: ${DB_DATABASE}
+            MYSQL_USER: ${DB_USERNAME}
+            MYSQL_PASSWORD: ${DB_PASSWORD}
+        volumes:
+           - ./storage/db:/var/lib/mysql
+networks:
```

Comment:
Maybe no need for the networking part here, it is a personal view.
Like here: https://github.com/nextcloud/docker/blob/master/.examples/docker-compose/insecure/mariadb-cron-redis/fpm/docker-compose.yml

Reply:
I have to disagree... i got this inspiration from mastodon:
https://github.com/tootsuite/mastodon/commit/9da81a16391edfcbda9c748dcd519fb3ebd765e5

Reply:
@lalop can I re-add the networks? :) please read the comments :point_up:

Reply:
Yes sure, it's your pr ;)